src/main/webapp/index.html

```diff
@@ -19,9 +19,13 @@
     <link rel="stylesheet" href="third-party/mdi-font/css/material-design-iconic-font.min.css">
     <link rel="stylesheet" href="third-party/select2/select2.min.css">
     <link rel="stylesheet" href="third-party/datepicker/daterangepicker.css">
+
+    <script src="https://apis.google.com/js/platform.js" async defer></script>
+    <meta name="google-signin-client_id" content="758286654746-8e5tr4b5tr0gukbkdjpb6vj6upd9pl6l.apps.googleusercontent.com">
 </head>
 
 <body>
+<<<<<<< HEAD
```

Comment:
there are version control markers in these files

we'll need to get rid of these

Reply:
done